tools/generate_hotkey.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+from pathlib import Path
+
+from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
+
+
+categories = {
+    category: {
+        item['help_text']: item['keys'] 
+        for item in KEY_BINDINGS.values()
+        if item['key_category'] == category
+    }
+    for category in HELP_CATEGORIES.keys()
+}
+OUTPUT_FILE = Path(__file__).parent.parent.joinpath('docs', 'hotkeys.md')
+
+with open(OUTPUT_FILE,"w") as mdFile:
+    mdFile.write("<!--- Generated automatically by tools/generate_hotkey.py -->\n"
```

Comment:
fstring?

Reply:
Since we are not using any variables here, do we need an fstring?

Reply:
The filename itself is a variable, so we can avoid adjusting this text (or part of it) when the script runs.

Reply:
I do not quite follow you here. Isn't the filename a constant with the root directory as zulipterminal. A similar case can be seen in `convert-unicode-emoji-data`.

Reply:
Noticed the change in `convert-unicode-emoji-data`, I've made the same change 👍🏼